in_toto/in_toto_verify.py

```diff
@@ -45,104 +70,34 @@
 from in_toto import verifylib
 from in_toto.models.metadata import Metablock
 
-
 # Command line interfaces should use in_toto base logger (c.f. in_toto.log)
 log = logging.getLogger("in_toto")
 
 
-def in_toto_verify(layout_path, layout_key_paths, layout_gpg_keyids, gpg_home):
-  """
-  <Purpose>
-    Loads the layout metadata as Metablock object (containing a Layout object)
-    and the signature verification keys from the passed paths and/or from
-    layout_gpg_keyids, calls verifylib.in_toto_verify
-    and handles exceptions.
-
-    The layout has to be signed by the private key corresponding to each passed
-    public key (path) or gpg key (keyid). If any of the signatures are missing
-    or invalid verification fails.
-
-  <Arguments>
-    layout_path:
-            Path to layout metadata file that is being verified.
-
-    layout_key_paths:
-            List of path(s) to project owner public key(s), used to verify the
-            root layout's signature(s).
-
-    layout_gpg_keyids:
-            List of project owner GPG keyid(s), used to verify the root
-            layout's signature(s).
-
-    gpg_home:
-            Path to GPG keyring (if not set the default keyring is used).
-
-  <Exceptions>
-    SystemExit if any exception occurs
-
-  <Side Effects>
-    Calls sys.exit(1) if an exception is raised
-
-  <Returns>
-    None.
-
-  """
-  try:
-    log.info("Verifying software supply chain...")
-
-    log.info("Loading layout...")
-    layout = Metablock.load(layout_path)
-
-    layout_key_dict = {}
-    if layout_key_paths != None:
-      log.info("Loading layout key(s)...")
-      layout_key_dict.update(
-          in_toto.util.import_rsa_public_keys_from_files_as_dict(
-          layout_key_paths))
-
-    if layout_gpg_keyids != None:
-      log.info("Loading layout gpg key(s)...")
-      layout_key_dict.update(
-          in_toto.util.import_gpg_public_keys_from_keyring_as_dict(
-          layout_gpg_keyids, gpg_home=gpg_home))
-
-    verifylib.in_toto_verify(layout, layout_key_dict)
-
-  except Exception as e:
-    log.info("{0} - {1}".format(type(e).__name__, e))
-    sys.exit(1)
 
 def main():
   """Parse arguments and call in_toto_verify. """
   parser = argparse.ArgumentParser(
       description="Verifies in-toto final product")
 
-  # Whitespace padding to align with program name
-  lpad = (len(parser.prog) + 1) * " "
+  parser.usage = "%(prog)s <named arguments> [optional arguments]"
 
-  parser.usage = ("\n"
-      "%(prog)s --layout <layout path>\n{0}"
-               "{{--layout-keys <filepath>[ <filepath> ...], "
-               " --gpg <keyid> [ <keyid> ...]}} \n{0}"
-               "[--gpg-home <path to gpg keyring>]\n{0}"
-               "[--verbose | --quiet]\n\n"
-               .format(lpad))
+  named_args = parser.add_argument_group("required named arguments")
```

Comment:
Ok, will change.

Reply:
Actually, I think I'll leave it as is. Because the `<required named arguments>` section doesn't list required positional arguments, which I would expect if it were called `<required arguments>`.
As for the variable name, `required_named_args` would probably be the best, but that seems a little long.